pkg/xray/resource_xray_report_test.go

```diff
@@ -41,33 +41,110 @@ var opRisksFilterFields = map[string]interface{}{
 	},
 }
 
-var violationsFilterFields = map[string]interface{}{
-	"filters": map[string]interface{}{
-		"type":         "security",
-		"watch_names":  []interface{}{"NameOfWatch1", "NameOfWatch2"}, // Conflicts with 'watch_patterns'
-		"component":    "*vulnerable:component*",
-		"artifact":     "some://impacted*artifact",
-		"policy_names": []interface{}{"policy1", "policy2"},
-		"severities":   []interface{}{"High", "Medium"},
-		"updated": map[string]interface{}{
-			"start": "2020-06-29T12:22:16Z",
-			"end":   "2020-07-29T12:22:16Z",
+var violationsFilterFields = []map[string]interface{}{
+	{
+		"filters": map[string]interface{}{
+			"type":         "security",
+			"watch_names":  []interface{}{"NameOfWatch1", "NameOfWatch2"}, // Conflicts with 'watch_patterns'
+			"component":    "*vulnerable:component*",
+			"artifact":     "some://impacted*artifact",
+			"policy_names": []interface{}{"policy1", "policy2"},
+			"severities":   []interface{}{"High", "Medium"},
+			"updated": map[string]interface{}{
+				"start": "2020-06-29T12:22:16Z",
+				"end":   "2020-07-29T12:22:16Z",
+			},
+			"security_filters": map[string]interface{}{
+				//"cve":      "CVE-2020-10693",
+				"issue_id": "XRAY-87343",
+				"cvss_score": map[string]interface{}{ // Conflicts with 'cve'
+					"min_score": 6.3,
+					"max_score": 9,
+				},
+				"summary_contains": "kernel",
+				"has_remediation":  true,
+			},
+			"license_filters": map[string]interface{}{
+				"unknown":       false,
+				"unrecognized":  true,
+				"license_names": []interface{}{"Apache", "MIT"}, // conflicts with license_patterns
+				//"license_patterns": []interface{}{"*Apache*", "The Apache*"},
```

Comment:
Is this still needed?

Reply:
I'd say yes. Let's keep it.

Reply:
I've removed all commented-out attributes, and kept all permutations in the test.